documentation/src/main/asciidoc/topics/data_container/proc_setting_cache_size.adoc

```diff
@@ -0,0 +1,23 @@
+[id='setting_cache_size_{context}']
+= Setting the Cache Size
+
+Setting the cache size configures the limit at which {brandname} begins eviction.
+
+[TIP]
+====
+Try different cache sizes to find optimal performance. If the cache size is to large, you can run out of memory. If it is too small, available memory is wasted.
+====
+
+Do one of the following:
+
+* For `COUNT`, set the maximum number of entries the cache can hold.
+* For `MEMORY`, set the maximum number of bytes the cache can take from memory.
+
+Declaratively::
```

Comment:
Again, there should be two procedures, or this info should be in the introduction.